Bot.py

```diff
@@ -73,18 +84,52 @@ def _connectSIO(self) -> None:
         """Sets up and connects the socket IO client
         """
         self.sio = socketio.Client()
-        self.sio.register_namespace(SocketIONamespace(self, "/"))
-        self.sio.connect("https://api.openmhz.com/")
+
+        # Register connect handler
+        self.sio.on("connect", self._connectHandler)
+        # Register disonnect handler
+        self.sio.on("disconnect", self._disconnectHandler)
+        # Register message handler because decorators are borked, and further using function names is borked
+        self.sio.on("new message", self._callHandler)
+
+        self.sio.connect("https://api.openmhz.com/", namespaces=["/"])
         self.sio.wait()
 
+    def _connectHandler(self) -> None:
+        """Connect event handler, sends start packet.
+        """
+        # Tell socketIO to send us data.
+        self.sio.emit("start", self.CONFIG)
+        log.info("Connected to socket")
+
+    def _disconnectHandler(self) -> None:
+        """Disconnect handler
+        """
+        log.info("Disconnected")
+
     def _kill(self, rec, frame) -> None:
         """This kills the c̶r̶a̶b̶  bot."""
         log.info("SIGINT or Ctrl-C hit. Exitting.")
         self.sio.emit("stop")
         self.sio.disconnect()
         exit(0)
 
-    def postTweet(self, call: dict):
+    def _callHandler(self, data):
+        """Message handler
+
+        Args:
+            data (str): The call data in a str
+        """
+        # See, here's why we needed to pass in our own reference and everything about this hurts me.
+        jsonData = json.loads(data)
+        self._postTweet(jsonData)
+
+        if self.reportLatency:
+            sum = sum(self.latency).total_seconds()
```

Comment:
I might suggest naming this something other than `sum` because `sum` is also a function name